layouts/partials/synthetics/network-layers.html

```diff
@@ -0,0 +1,33 @@
+{{ $dot := . }}
+<div class="logs-containers">
+    <div class="container cards-dd">
+        <div class="card-deck">
+            <a class="card" href="/api_tests/http_tests">
```

Comment:
this link is 404'ing for me - maybe missing the `/synthetics/` ?

Reply:
Great catch!

```suggestion
            <a class="card" href="/synthetics/api_tests/http_tests">
```